app/src/main/java/app/foodfinderapp/MainActivity.kt

```diff
@@ -60,7 +58,7 @@ class MainActivity : AppCompatActivity() {
         }
 
         adapter = RestaurantAdapter(restaurantList)
-        recyclerView = findViewById<RecyclerView>(R.id.restaurantRecyclerView)
+        recyclerView = findViewById(R.id.restaurantRecyclerView)
```

Comment:
it is better omit explicit words so that code is more readabile and if it does not add value as it is not nessassary always to state everything explicitly